haystack/nodes/prompt/prompt_node.py

```diff
@@ -362,15 +361,13 @@ def invoke(self, *args, **kwargs):
 
     @classmethod
     def supports(cls, model_name_or_path: str) -> bool:
-        if not all(m in model_name_or_path for m in ["google", "flan", "t5"]):
```

Comment:
In the retro, we discussed that we should raise a warning saying, "We haven't tested this model" whenever someone tries to load a non-flan model.

@vblagoje Can you please suggest what msg you would like here?

Reply:
PromptNode initialized with a non-instruction following language model. A wide range of PromptTemplates will likely not work as intended; please use custom prompts and PromptTemplates specific for {insert-model-name} model